edbterraform/data/terraform/aws/modules/specification/variables.tf

```diff
@@ -23,11 +38,15 @@ variable "spec" {
         port        = optional(number)
         protocol    = string
         description = string
+        ingress_cidrs = optional(list(string), ["0.0.0.0/0"])
```

Comment:
Is it a default value? Why do we need it here and we don't have for the others `ingress/egress_cidrs`?

Reply:
It is only set under `service_ports` since we assume they are meant to be accessible to all unless the user specifies otherwise, so it doesnt change from the behavior we had before.

Anything that is left as null will be set to the region_cidrs variable since the module handles null values with a default.
https://github.com/EnterpriseDB/edb-terraform/pull/34/files#diff-ffb3ff213ecbd7a5a85f96b88e9fa436620b45096522bfa760c89fbc3dadbd54R54-R56